metaci/templates/admin/release/release/change_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls static admin_modify %}
+{% block submit_buttons_bottom %}
+{% submit_row %}
+{% if METACI_RELEASE_WEBHOOK_URL and METACI_START_STOP_WEBHOOK == "True" %}
```

Comment:
I think this can just be `if METACI_RELEASE_WEBHOOK_URL` ... being able to create the change case can still be useful even if the option to do start/stop for implementation steps isn't enabled.